src/components/text-area/text-area.tsx

```diff
@@ -0,0 +1,476 @@
+import {
+  Component,
+  h,
+  Prop,
+  Element,
+  Event,
+  EventEmitter,
+  VNode,
+  Watch,
+  Method,
+  Host,
+  State
+} from "@stencil/core";
+import { connectForm, disconnectForm, FormComponent, HiddenFormInputSlot } from "../../utils/form";
+import { connectLabel, disconnectLabel, getLabelText, LabelableComponent } from "../../utils/label";
+import { slotChangeHasAssignedElement, toAriaBoolean } from "../../utils/dom";
+import { CSS, SLOTS, RESIZE_TIMEOUT } from "./resources";
+import {
+  connectLocalized,
+  disconnectLocalized,
+  LocalizedComponent,
+  NumberingSystem,
+  numberStringFormatter
+} from "../../utils/locale";
+import { createObserver } from "../../utils/observers";
+import {
+  componentLoaded,
+  LoadableComponent,
+  setComponentLoaded,
+  setUpLoadableComponent
+} from "../../utils/loadable";
+import {
+  connectMessages,
+  disconnectMessages,
+  setUpMessages,
+  T9nComponent,
+  updateMessages
+} from "../../utils/t9n";
+import { TextAreaMessages } from "./assets/text-area/t9n";
+import { throttle } from "lodash-es";
+import { InteractiveComponent, updateHostInteraction } from "../../utils/interactive";
+
+/**
+ * @slot - A slot for adding text.
```

Comment:
For default slots and consistency we tried to mention content, not sure if that's applicable here, but WDYT about:

> A slot for adding content.

Reply:
content sounds better.

Reply:
i think it is mentioned as `text` for some components in the cc repo ( Ex: chip, button, link, input-text). i will leave it up to you @geospatialem

Reply:
Ah, yes, in those cases we're trying to be focused on text, which might be the case here as well. Let's leave as-is for consistency with the other components. Thanks @anveshmekala!